docs/docsite/rst/user_guide/playbooks_tags.rst

```diff
@@ -39,6 +39,8 @@ tasks, you can use the ``--skip-tags`` command-line option::
 
     ansible-playbook example.yml --skip-tags "packages"
 
+.. warning::
+    * When ``--tags`` or ``--skip-tags`` is supplied to the playbook, fact gathering about remote hosts is skipped. The ``tags`` can be only used in the plays and below.
```

Comment:
still incorrect, fact gathering is tagged with 'always' by default, it is ONLY skipped if you apply a tag and then use a different tag in ``--tags`` or the same tag in ``--skip-tags``.

Reply:
@bcoca ! Updated.